doc/ansible/dev/testing_guide.md

```diff
@@ -3,12 +3,13 @@
 ## Getting started
 
 ### Requirements
-To begin, you sould have:
+To begin, you should have:
 - The latest version of the [operator-sdk](https://github.com/operator-framework/operator-sdk) installed.
-- Docker installed and running
+- [Docker](https://www.docker.com/get-started) installed and running
 - [Molecule](https://github.com/ansible/molecule) >= v2.20
 - [Ansible](https://github.com/ansible/ansible) >= v2.7
-- [jmespath](https://pypi.org/project/jmespath/)
+- [JMESPath](https://github.com/jmespath/jmespath.py)
+- [The Docker Python Library](https://github.com/docker/docker-py)
```

Comment:
I am not sure about `Docker Python Library`. I do no really think that it is required. 
@fabianvf wdyt?

Reply:
that is required to spin up the dockerized cluster. I believe it used to be pulled in on molecule install, we should probably update the molecule install instructure to have users run `pip install molecule[docker]` which pulls in those dependencies.